client/jetpack-cloud/sections/comparison/table/style.scss

```diff
@@ -322,7 +322,7 @@
 		width: 200px;
 	}
 
-	col.product-jetpack-starter {
+	col.product-jetpack-growth {
```

Comment:
Did you mean to replace this or add another?

Reply:
I meant to replace it. Starter is deprecated and isn't used on this page anymore, but I wanted to have the same styles as it, as it fit well with the others